lib/fileSystem.js

```diff
@@ -0,0 +1,115 @@
+var q = require('q');
+
+/**
+ * @param {object} e error object
+ */
+function _fsErrorHandler(e) {
+  var msg = '';
+
+  switch (e.code) {
+  case FileError.QUOTA_EXCEEDED_ERR:
```

Comment:
This would crash on modern webview. See: https://github.com/brianleroux/lawnchair/pull/229